src/page/stableMatching/Processing.jsx

```diff
@@ -210,8 +262,17 @@ export default function InputProcessingPage() {
 
         {/* Lựa chọn Problem type */}
         <div className="problem-type-chooser">
-          <p className="problem-type-text bold">Choose a problem type: </p>
```

Comment:
{
            isParallel && (
                  <i>
                    RBO driver currently available for Many to Many RBO only
                  </i>
              )
          }
          
          
          bỏ đi